### ISSUE TITLE
training result

### ISSUE THREAD

**Issue by LY1712** [Deep-Learning-Person-Re-Identification/Implementaion-1#40]
In training, the result is that the precision of the training set reaches 99 or more, but the accuracy of the validation set is only 56. The loss values also vary greatly, there is an overfitting situation, and the first rank-1 shows 0.08. It is how it happened? @Ning-Ding @prashanthbasani @wangbm
